Sources/Conduit/Networking/Images/ImageDownloader.swift

```diff
@@ -18,9 +18,13 @@ public enum ImageDownloaderError: Error {
     case invalidRequest
 }
 
+public protocol ImageDownloading {
```

Comment:
I think we have a naming convention of `{{Subject|Action}}Type` or simply `{{AgentName}}Type` for protocols of the agents.

In this case `ImageDownloaderType`

Reply:
💯